### PR TITLE
Fix bug that causes diff in license.go file when building daemon scheduler

### PR DESCRIPTION
**Pull request by tuedtran**
#### Summary
Fix a bug that causes diff in license.go file when building daemon scheduler.
Related issue: https://github.com/blox/blox/issues/80

#### Implementation details
_ Set environment language to C to keep sorting consistent across different environments.
_ Strip out Windows-styled end-of-line character CRLF when generating license.go.
_ Make generator scripts for both DS and CSS identical.

#### Testing

- [x] cluster-state-service binary built locally and unit-tests pass (`cd cluster-state-service; make; cd ../`)
- [x] cluster-state-service build in Docker succeeds (`cd cluster-state-service; make release; cd ../`)
- [x] daemon-scheduler binary built locally and unit-tests pass (`cd daemon-scheduler; make; cd ../`)
- [x] daemon-scheduler build in Docker succeeds (`cd daemon-scheduler; make release; cd ../`)

New tests cover the changes: no 

#### Description for the changelog

Fix a bug that causes diff in license.go file when building daemon scheduler.
Related issue: https://github.com/blox/blox/issues/80

#### Licensing

This contribution is under the terms of the Apache 2.0 License: Yes

### CODE REVIEW

**Comment by samuelkarp** [comment]
> Remove Windows-styled end-of-line character CRLF from LICENSE.txt in package cihub/seelog.

It's a bad practice to change a vendored source file.  It introduces a change from upstream which is not tracked anywhere but here and it will get blown away next time the dependency gets updated.  It would be better to change the generator to strip out CRLF from all the license files instead.

**Comment by kylbarnes** [comment]
I agree with @samuelkarp. We shouldn't change the actual vendor source, as doing a fresh pull would revert our local change. Can you strip out windows carriage returns in our shell script?

**Comment by samuelkarp** [comment]
@tuedtran Your new commit is still manually changing the line-endings, so the next time someone runs `make generate` your changes might be reverted.  Can you adjust the generator itself to pick a consistent line-ending?

**Comment by tuedtran** [comment]
Adjusted the generator to remove Carriage Returns from file as suggested. Also made both generators from CSS and DS identical.

File: daemon-scheduler/scripts/license.sh [hunk 1]
```diff
@@ -52,6 +55,7 @@ for repo in ./../../vendor/gopkg.in/* ./../../vendor/google.golang.org/*; do
 done;
 
 inputfile+="
+***
```
Comment (narehayrapetyan) [inline daemon-scheduler/scripts/license.sh]: what's this?

File: licenses/license.go [hunk 2]
```diff
@@ -6049,6 +6049,7 @@ THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+***
```
Comment (narehayrapetyan) [inline licenses/license.go]: here as well

File: daemon-scheduler/scripts/license.sh (hunk 1 above)
Comment (kylbarnes) [inline daemon-scheduler/scripts/license.sh]: `***` is the space separator between individual licenses in license.go.

Comment (narehayrapetyan) [inline daemon-scheduler/scripts/license.sh]: oh I see. Thanks, @kylbarnes

**Comment by tuedtran** [comment]
I've tested it with multiple environments, it worked fine.
Since I can't merge this myself, whoever does please squash these commits before merging.

File: cluster-state-service/scripts/license.sh [hunk 3]
```diff
@@ -76,7 +79,7 @@ CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 "
 
-cat << EOF > "${outputfile}"
+cat << EOF | tr -d '\r' > "${outputfile}"
```
Comment (dbenhur) [inline cluster-state-service/scripts/license.sh]: -1 for [gratuitous use of cat](http://porkmail.org/era/unix/award.html#cat).

Comment (tuedtran) [inline cluster-state-service/scripts/license.sh]: Issue fixed: https://github.com/blox/blox/pull/121

**Pull request merged**